decko/lib/decko/tasks/decko/bootstrap.rake

```diff
@@ -1,124 +1,146 @@
 namespace :decko do
   namespace :bootstrap do
-    desc "rid template of unneeded cards, acts, actions, changes, " \
-         "and references"
+    desc "reseed, re-clean, and re-dump"
+    task update: :environment do
+      %w(reseed bootstrap:clean bootstrap:dump).each do |task|
+        Rake::Task["decko:#{task}"].invoke
+      end
+    end
```

Comment:
I started working on this because I wanted to get new decks running without having to update machine output.  got into a bit further than intended.